src/components/Editor/Editor.css

```diff
@@ -98,7 +98,7 @@ html[dir="rtl"] .editor-mount {
 }
 
 .highlight-line .CodeMirror-line {
-  animation: fade-highlight-out 1.5s normal forwards;
+  animation: fade-highlight-out 3s normal forwards;
```

Comment:
Hi @jasonLaster !

1. Clicking on a breakpoint in the breakpoint list would trigger `highlightLine()`, but the function returns right away, and it doesn't execute `this.editor.codeMirror.addLineClass(line - 1, "line", "highlight-line");`, which is highlighting the line.
![yvxe4w1ual](https://cloud.githubusercontent.com/assets/4562118/22633002/ce8c8f42-ec58-11e6-8efe-754017257f96.gif)




2. Selecting a frame in the call stack would trigger `highlightLine()` and clear the line, but it doesn't call `this.editor.codeMirror.addLineClass(line - 1, "line", "highlight-line");`, so we won't see the line being applied with the styles of `.highlight-line .CodeMirror-line`.
![2ped7795ob](https://cloud.githubusercontent.com/assets/4562118/22632946/5deb98a0-ec58-11e6-9558-c14fa782f06a.gif)